app/services/idv/agent.rb

```diff
@@ -11,7 +11,7 @@ def proof_resolution(
       user_id:,
       threatmetrix_session_id:,
       request_ip:,
-      double_address_verification: false
+      ipp_enrollment_in_progress: false
```

Comment:
leave this rename until the next PR otherwise old boxes that don't know the name `ipp_enrollment_in_progress` will error

Reply:
@zachmargolis thank you for catching this!